Makefile

```diff
@@ -40,7 +55,7 @@ tail-logs: ## Tail the logs for the project service containers (Filtered via SER
 # For deploying
 
 pull-latest:
-	$(if $(APP_ENV_IS_PROD), git checkout master && git pull, $(info -- Not pulling on $(APP_ENV) environment))
+	$(if $(APP_ENV_IS_STAGE_OR_PROD), git checkout master && git pull, $(info -- Not pulling on $(APP_ENV) environment))
```

Comment:
@suvash 

I have typically been deploying the `dev` branch to the test/staging server. Then, once it looks right, I have been merging `dev` into `master` and deploying `master` to production. Any thoughts about whether or not it makes sense to be able to checkout a specific branch here versus always checking out master?

Reply:
Ah, sure then. I can get rid of this make target altogether. That way, git branch workflows can be handled outside of the make targets. Keeps things simple as well.